components/templates/Cargo.toml

```diff
@@ -12,6 +13,11 @@ toml = "0.4"
 csv = "1"
 serde_json = "1.0"
 error-chain = "0.12"
+crypto-hash = "0.3.1"
```

Comment:
we don't need a crypto hash, we can use the one from std (`std::hash::Hash`) like the imageproc sub-crate does

Reply:
ah, i somehow missed the implementation for `String` :+1: